### DISCUSSION TITLE
How to configure report when running user flows?

### DISCUSSION THREAD

**Discussion by kskoeld** [GoogleChrome/lighthouse#14095] [Q&A]
I'm running automated user flows using the approach [as described in the docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md#complete-user-flow-code)
However I'm only interested in the performance part of the report.
How can I use a `custom-config.js` file for the user flows which for example looks like this?
`module.exports = {
  extends: 'lighthouse:default',
  settings: {
    onlyCategories: [
      'performance'
    ],
  },
};`

Or is there a different way to customize the report configuration?

**Comment by adamraine** [comment]
You will need to pass the config as a parameter to `startFlow`:

```js
const config = {
  extends: 'lighthouse:default',
  settings: {
    onlyCategories: ['performance'],
  },
}
const flow = await api.startFlow(page, {config});
```
